playwright/plugins/docker/index.ts

```diff
@@ -45,12 +45,14 @@ export class Docker {
             }
         }
 
+        // Avoid using `--rm` in CI as if a container crashes this means its logs are wiped out
+        if (!process.env.CI) params.unshift("--rm");
```

Comment:
I spent a long time assuming this meant we are *removing* the `--rm` parameter. Basically, the comment is useful, but not describing what the code is actually doing.

```suggestion
        // Provided we are not running in CI, add a `--rm` parameter.
        // There is no need to remove containers in CI (since they are automatically removed anyway), and
        // `--rm` means that if a container crashes this means its logs are wiped out.
        if (!process.env.CI) params.unshift("--rm");
```


... I'm slightly unconvinced by the rather magical, undocumented behaviour of `CI` changing behaviour here, but I'll live with it.